app/components/works/edit/contributor_component.html.erb

```diff
@@ -5,16 +5,16 @@
                                          field_name: :role_type,
                                          label: 'Individual',
                                          value: 'person',
-                                         label_data: { action: 'click->contributors#showPersonSection' }) %>
+                                         label_data: { action: 'click->contributors#showPersonSection click->tooltips#hideAll' }) %>
```

Comment:
Whenever a form interaction swaps out some content for other content, hide open tooltips.